diff --git a/retroarch.c b/retroarch.c
index dab91c2..019f85e 100644
--- a/retroarch.c
+++ b/retroarch.c
@@ -1571,6 +1571,13 @@ void drivers_init(
          hwr->context_reset();
       video_st->flags            &= ~VIDEO_FLAG_CACHE_CONTEXT_ACK;
       runloop_st->frame_time_last = 0;
+
+#ifdef HAVE_SPRUCE_IGM_SW
+      {
+         FILE *f = fopen("/tmp/spruce_loading_done", "w");
+         if (f) fclose(f);
+      }
+#endif
    }
 
    /* Regular display refresh rate startup autoswitch based on content av_info. */
